### DISCUSSION TITLE
Dumb Question: Is the client->server password stored in a flat file somewhere?

### DISCUSSION THREAD

**Discussion by BigWincy** [Nimdy/Dedicated_Valheim_Server_Script#48] [Q&A]
Dumb Question: Is the client->server password stored in a flat file somewhere?

I searched all through /etc/ and /home/steam/.config/, but couldn't find a config file. I'd like the capability to change the client password without reinstalling the server each time. I figure the password would just be stored in a config file. I'm sure a stop, change, start procedure is required.

Passwords can be compromised, especially when kids have them. I think we should all have the procedure down as a good security practice.

Also, on a side note: I looked through the logs to see if I could find bad logon attempts to detect brute force attacks. There's a log called connections, but it looks like a server-to-steam connection. As the game gets more popular, nefarious actors will starting targeting us more and more. It'd be nice to see if you are getting hit, ahead of time.

Make strong client passwords, back up your worlds, & Stay Safe,
- Wincy

**Comment by Nimdy** [comment]
server password is stored with the valheim start . sh file and now if you build it with the script is stored for historic  /home/steam/serverSetup.txt .

I would look at this just as we all did with Minecraft before it blew up... just gotta go with it.

In the future... maybe something can be done but right now... just make the password something you dont mind.

This game was not designed to be large scale... its peer-2-peer.... maybe in the future it will change and then the folks  like you and I can make different functions.... This password needs to be stored some where... but I hear you and we are all limited to IronGates scripting requirements.

This is why my new script I did something different.... I made a menu system (use ./menu and not ./build....)  that you can use to install your server with.  The menu system forces you to use a upper...lower... and number (without special chars) for your server.  This is the best I can do to help security until much later!

GLHF!

**Comment by Nimdy** [comment]
:)
